shopping-cart/shopping-cart-java/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.4.2
```

Comment:
I tried bumping to 1.4.9 but there was some corrupted cache on travis and sbt 1.4.9 continually failed. Using 1.3.13 aligns the version with all other samples.